app/src/main/res/layout/profile_chooser_fragment.xml

```diff
@@ -63,41 +90,38 @@
       </androidx.constraintlayout.widget.ConstraintLayout>
     </androidx.core.widget.NestedScrollView>
 
-    <LinearLayout
+    <View
       android:layout_width="match_parent"
-      android:layout_height="128dp"
-      android:layout_gravity="bottom"
-      android:background="@drawable/admin_controls_gradient"
-      android:gravity="bottom"
-      android:orientation="horizontal"
-      android:paddingStart="20dp"
-      android:paddingEnd="20dp"
-      android:paddingBottom="24dp">
-
-      <LinearLayout
-        android:id="@+id/administrator_controls_linear_layout"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:onClick="@{(v) -> viewModel.onAdministratorControlsButtonClicked()}"
-        android:orientation="horizontal">
+      android:layout_height="96dp"
+      android:background="@drawable/profile_chooser_gradient"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent" />
 
-        <TextView
-          android:layout_width="0dp"
-          android:layout_height="wrap_content"
-          android:layout_marginEnd="8dp"
-          android:layout_marginBottom="2dp"
-          android:layout_weight="1"
-          android:fontFamily="sans-serif-medium"
-          android:gravity="end"
-          android:text="@string/administrator_controls"
-          android:textColor="@color/oppiaProgressChapterNotFinished"
-          android:textSize="14sp" />
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="24dp"
+      android:fontFamily="sans-serif"
```

Comment:
In mock font-family is Helvetica Neue. Did you clarify this with @mschanteltc .

Reply:
Ya it's sans-serif.